### PR TITLE
Added sjfs

### LINKED ISSUE #3988: Shortest Job First Scheduling Code in Operating Systems
**This is a(n):**

- [x ] New algorithm
- [ ] Update to an existing algorithm
- [ ] Error
- [ ] Proposal to the Repository

**Details:**

 Shortest job first (SJF) or shortest job next, is a scheduling policy that selects the waiting process with the smallest execution time to execute next. SJN is a non-preemptive algorithm

### PR DESCRIPTION
**Pull request by aSquare14**
**Fixes issue:**

Closes #3988 

**Changes:**

Added a SJFS folder in operating system and c++ implementation of the same.

### CODE REVIEW

File: code/operating_system/src/scheduling/shortest_job_first_ scheduling/shortest_job_first_ scheduling.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// C++ program to implement Shortest Job first 
+
+#include<bits/stdc++.h>
```
Comment (arnavb) [inline code/operating_system/src/scheduling/shortest_job_first_ scheduling/shortest_job_first_ scheduling.cpp]: #1915.

File: code/operating_system/src/scheduling/shortest_job_first_ scheduling/shortest_job_first_ scheduling.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
+// C++ program to implement Shortest Job first 
+
+#include<bits/stdc++.h> 
+using namespace std;
```
Comment (arnavb) [inline code/operating_system/src/scheduling/shortest_job_first_ scheduling/shortest_job_first_ scheduling.cpp]: Don't use this.

**Comment by aSquare14** [comment]
@AdiChat Please Review.

**Comment by aSquare14** [comment]
@AdiChat  Please review

**Comment by arnavb** [comment]
Please remove using namespace std from your code and follow our C++ style guide.

**Pull request closed without merging**